### ISSUE TITLE
misleading settings in README

### ISSUE THREAD

**Issue by zeyu-liu** [open-mmlab/mmaction2#178]
I got the suggestions "The gpus indicates the number of gpu we used to get the checkpoint. It is noteworthy that the configs we provide are used for 8 gpus as default. According to the Linear Scaling Rule, you may set the learning rate proportional to the batch size if you use different GPUs or videos per GPU, e.g., lr=0.01 for 4 GPUs * 2 video/gpu and lr=0.08 for 16 GPUs * 4 video/gpu." when tried to use slowfast configs.

Yet the lr and videos_per_gpu in these configs files are different from those in README. For example, in https://github.com/open-mmlab/mmaction2/blob/master/configs/recognition/slowfast/slowfast_r50_4x16x1_256e_kinetics400_rgb.py, the 'lr' is 0.1 and the 'videos_per_gpu' is 8. 

So, which one is the correct setting to reproduce the performance mentioned in README?

**Comment by innerlee** [comment]
both are right.
we recompute the lr for 8gpus when writing the config. users can also compute the lr used in the ckpt by the lr rules

**Comment by zeyu-liu** [comment]
> both are right.
> we recompute the lr for 8gpus when writing the config. users can also compute the lr used in the ckpt by the lr rules

OK, I found this in the training log. Thanks.
